### PR TITLE
MSDC July 2023 updates

### PR DESCRIPTION
**Pull request by lorieide**
Overview for July 2023 updates

### CODE REVIEW

**Comment by lorieide** [comment]
The broken image links have been fixed (lines 21-33). But for the issue of the absolute links, they are working but will need to be updated to relative links.

File: concepts/data-connect-concept-overview.md
Comment (kestrel144) [inline concepts/data-connect-concept-overview.md:1]: Line 40 - change M365 to Microsoft 365
Line 41 - should Basic Datasets be lowercase? It's lowercased in line 40
Line 46 - These terms were lowercased in lines 23-26:   Customer Relationship Analytics, Business Process Analytics, Security and Compliance Analytics, and People Productivity Analytics. They need to be lowercase here or uppercase there.

Comment (lorieide) [reply concepts/data-connect-concept-overview.md:1]: I incorporated these fixes.

**Comment by gladyscodes** [comment]
@lorieide I was reviewing and in the preview it looks like the LotisBlue logo is not a png file. are you seeing the same? thanks!

**Comment by rimisra2** [comment]
@lorieide  can you please ensure that the images are rendering as expected. They are not included in the PRs and are part of the warning.

**Comment by lorieide** [comment]
> @lorieide I was reviewing and in the preview it looks like the LotisBlue logo is not a png file. are you seeing the same? thanks!

Thanks for letting me know. It should be fixed now.

**Comment by lorieide** [comment]
> @lorieide can you please ensure that the images are rendering as expected. They are not included in the PRs and are part of the warning.

I fixed the images and also the absolute links are now relative links. Thanks

**Pull request closed without merging**